person.rb

```diff
@@ -0,0 +1,22 @@
+class Person
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_reader :id
```

Comment:
[OPTIONAL]

- For best practices, always use `attr_accessor` and `attr_reader` on the top, just below the class.

Reply:
Hello @Dipeshtwis , Thank you for your valuable comment, I changed the position of attr_reader to be on the top as u can see in the image below:
![Capture5](https://user-images.githubusercontent.com/60292761/140779463-10b33fb9-3b42-4476-a452-524c41688958.PNG)